test/ILLink.RoslynAnalyzer.Tests/TestCaseUtils.cs

```diff
@@ -119,6 +120,13 @@ public static void GetDirectoryPaths (out string rootSourceDirectory, out string
 			testAssemblyPath = Path.GetFullPath (Path.Combine (artifactsBinDir, "ILLink.RoslynAnalyzer.Tests", configDirectoryName, tfm));
 		}
 
+		// Accepts typeof expressions, with a format specifier
+		public static string GetStringFromExpression (TypeOfExpressionSyntax expr, SemanticModel semanticModel, SymbolDisplayFormat displayFormat)
+		{
+			var typeSymbol = semanticModel.GetSymbolInfo (expr.Type).Symbol;
+			return typeSymbol?.ToDisplayString (displayFormat) ?? throw new InvalidOperationException ();
+		}
```

Comment:
Should we add `displayFormat` as an optional parameter to the bellow method and put these two lines there?

Reply:
I made them separate because I don't think there's an obvious way to give a string representation of an expression in general. I'd rather limit the below method to expressions which are building up strings from literals, and have separate variants used to format types (or maybe other kinds in the future).